mzLib.sln

```diff
@@ -166,12 +168,6 @@ Global
 		{AD596BA0-702A-45CB-B9EB-F50D78259FC8}.Release|Any CPU.Build.0 = Release|Any CPU
 		{AD596BA0-702A-45CB-B9EB-F50D78259FC8}.TestAndRelease|Any CPU.ActiveCfg = Release|Any CPU
 		{AD596BA0-702A-45CB-B9EB-F50D78259FC8}.TestAndRelease|Any CPU.Build.0 = Release|Any CPU
-		{EDF2377D-9FA8-4F85-A3E5-63FE085F956F}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
I think I actually was wrong about this, it looks like all the other projects have these configs?

Reply:
That would make sense why they keep getting made